systemd/org.freedesktop.secrets.service

```diff
@@ -0,0 +1,4 @@
+[D-BUS Service]
+Name=org.freedesktop.secrets
+Exec=/bin/false
+SystemdService=dbus-org.freedesktop.secrets.service
```

Comment:
Why is this "dumb" service needed? [not a systemd wizard here]

Reply:
This is the part that lets dbus automatically start the actual service when a client tries to talk to `org.freedesktop.secrets`.  Without this unit installed on the system, applications can still talk to the service, but it must first be started by some other means (manually, by a target, etc.).  Would you like me to add this explanation to the `README`?

Reply:
Does this mean a user will start his service for the system? I'm wondering if this unit should not also be scoped to the user.

Reply:
No, it starts it for the user because the service is installed as a user service (assuming it is installed in the `user` systemd directory, as specified in the installation instructions as part of this PR).